liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseLauncher.java

```diff
@@ -122,4 +131,13 @@ private static void debug(String message, Throwable e) {
             }
         }
     }
+
+    private static String getValueFromJvmPropertyOrEnv(String name) {
+        String value = System.getProperty(name);
```

Comment:
Generally system properties are more in the format `liquibase.home` or `liquibase.launcher.debug`

Can you translate the env varible format to the other and check for it as a system property too?

Reply:
It makes perfect sense to me, I agree with you.

Of course, I will implement the changes tonight.

Reply:
I updated the code to provide a more appropriate naming convention for the JVM properties.

Please, don't hesitate to contact me if further changes should be performed.